aasemble/django/apps/buildsvc/pkgbuild/python.py

```diff
@@ -41,7 +43,34 @@ def binary_pkg_name(self):
 
         return 'python-%s' % (pkgname,)
 
+    def get_pydist_overrides(self):
+        return self.get_aasemble_config().get('pydist_override', {})
+
+    def add_pydist_overrides(self):
+        override_dict = self.get_pydist_overrides()
+        if len(override_dict) == 0:
+            self.logger.info('No pydist-overrides in config')
+            return
+        pydist_overrides_str = (yaml.dump(override_dict, default_flow_style=False)).replace(':', '')
+        self.logger.info('pydist-overrides found in config: %s' % (pydist_overrides_str))
+        pydist_overrides = os.path.join(self.builddir, 'debian', 'pydist-overrides')
+
+        if os.path.exists(pydist_overrides):
+            with open(pydist_overrides, 'r') as fp:
+                pre_pydist_overrides = fp.read()
```

Comment:
Presently this is not the case - but will be adding support for debian folder + python builder capability.